src/libraries/System.Net.Sockets/tests/FunctionalTests/DualModeSocketTest.cs

```diff
@@ -2489,7 +2493,8 @@ public SocketServer(ITestOutputHelper output, IPAddress address, bool dualMode,
                     _server = new Socket(address.AddressFamily, SocketType.Stream, ProtocolType.Tcp);
                 }
 
-                port = _server.BindToAnonymousPort(address);
```

Comment:
Isn't BindToAnonymousPort still preferable to BindToPoolPort in non dualMode cases?

Reply:
The class is being used by several fragile DualMode cases. We could pass a parameter / use alternative constructor to distinguish them, but it doesn't worth the additional complexity IMO.